pkg/apis/k0s/v1beta1/api_test.go

```diff
@@ -35,7 +35,8 @@ func (s *APISuite) TestValidation() {
 
 	s.T().Run("accepts_ipv6_as_address", func(t *testing.T) {
 		a := APISpec{
-			Address: "2001:0db8:85a3:0000:0000:8a2e:0370:7334",
+			Address:     "2001:0db8:85a3:0000:0000:8a2e:0370:7334",
+			BindAddress: "2001:0db8:85a3:0000:0000:8a2e:0370:7334",
```

Comment:
Do we need to add the BindAddress to theexisting test cases?

Reply:
As in: should we leave them untouched?